packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker.modal.tsx

```diff
@@ -64,12 +64,29 @@ const PrintIdentifierSticker: React.FC<PrintIdentifierStickerProps> = ({ closeMo
     setIsPrinting(false);
   }, []);
 
+  const handleInitiatePrint = useCallback(
+    (printWindow: HTMLIFrameElement | null): Promise<void> => {
+      return new Promise<void>((resolve) => {
+        if (printWindow) {
+          const printContent = printWindow.contentDocument || printWindow.contentWindow?.document;
+          if (printContent) {
+            printContent.documentElement.style.setProperty('--print-scale', printScale);
```

Comment:
thank you @vasharma05, don't you think we should automate this? ie We scale basing on the sticker size and available window size.

Reply:
If it brings some level of complex, we can ignore it.

Reply:
I tried automating this, but it didn't work for me. Can you help me in the same?
Thanks!